pygac/reader.py

```diff
@@ -436,8 +438,8 @@ def to_datetime64(year, jday, msec):
             numpy.datetime64: Converted timestamps
 
         """
-        return (((year - 1970).astype('datetime64[Y]')
-                 + (jday - 1).astype('timedelta64[D]')).astype('datetime64[ms]')
+        return (year.astype(str).astype('datetime64[Y]')
```

Comment:
Could you explain the logic here?

Reply:
I usually try to avoid hard coded numbers as much as possible. In this case, I did not like that PyGAC hard codes the `numpy.datetime64` epoch (1970). I know that there is a very low change that numpy will ever change the epoch, but still I think PyGAC should not make any assumptions on it.
The shortest way I found to become independent of the epoch was the string conversion, because something like `epoch = int(str(np.datetime64(0, 'Y')))` looks ugly to me and numpy also does not have any attribute to access its epoch directly (or at least I could not easily find it).

Reply:
Makes sense, thanks for explaining!